src/nora/components/Tag/Tag.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './Tag.module.scss'
+
+const Tag = ({ children, type }) => {
+  let colorClass
+  switch (type) {
+    case 'approved':
```

Comment:
Pretty sure we'll need some more states, but this is good and we'll be iterative! 🙌